### PR TITLE
game

### PR DESCRIPTION
**Pull request by peterjuma**
- Add puzzle
- Add game

### CODE REVIEW

File: game.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Muzzle
```
[the remark this answers is not in the document]
Comment (jumapeter) [reply game.md:1]: Add comment to a review

[the remark this answers is not in the document]
Comment (jumapeter) [reply game.md:1]: Responding

**Review by pjuma — DISMISSED** [review]
Approve changes

**Review by pjuma — DISMISSED** [review]
Ok

**Pull request merged**